doc/test_messages_documentation.py

```diff
@@ -46,6 +49,12 @@ def get_functional_test_files_from_directory(input_dir: Path) -> List[Tuple[str,
                 suite.append(
                     (message_dir.stem, message_dir / "bad.py"),
                 )
+            if (message_dir / "related.rst").exists():
+                with open(message_dir / "related.rst", encoding="utf-8") as file:
+                    text = file.read()
+                    assert text.startswith(
+                        "-"
+                    ), f"{message_dir / 'related.rst'} should be a list using '-'."
```

Comment:
```suggestion
                    assert text.startswith(                        "-"                    ) or text.startswith(                        "*"                    ), f"{message_dir / 'related.rst'} should be a list using '-'."
```

Reply:
Why not require consistency now? We're not too far in to change it now